testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -497,125 +509,134 @@ def choosemeth(sel, meth, periodic):
 
         return sel
 
-    def _check_around(self, meth, periodic):
-        sel = Parser.parse('around 5.0 resid 1', self.u.atoms)
+    @pytest.mark.parametrize('u, meth, periodic', [
```

Comment:
Oh yes, when I was trying I had a `dec.skipif` and that doesn't work well with static methods. That's why I got different results.